website/pages/docs/commands/volume/register.mdx

```diff
@@ -91,7 +91,7 @@ context {
 - `mount_options` - Options for mounting `file-system` volumes that don't
   already have a pre-formatted file system. Consult the documentation for your
   storage provider and CSI plugin as to whether these options are required or
-  neccessary.
+  necessary.
```

Comment:
Note to reviewers: I dunno why these slipped by the docs PR but triggered the linter here. Maybe we don't check spelling there?